README.md

```diff
@@ -0,0 +1,81 @@
+## Termibbl
+
+A skribbl.io alike, but running in your terminal.
+
+Created in Rust
+
+### Installation
+
+### Source
+
+#### Nix
+```
+git clone https://github.com/elkowar/Termibbl
+cd Termibbl
+nix build
+```
```

Comment:
```suggestion
```sh
git clone https://github.com/elkowar/Termibbl
cd Termibbl
nix build
```
```
This applies to all of the code blocks ;D